scripts/trivia.js

```diff
@@ -2158,6 +2191,10 @@ addAdminCommand(["search"], function (src, commandData, channel) {
 }, "Allows you to search through the questions, format /search [query]. Only matches whole words.");
 
 addAdminCommand(["apropos"], function (src, commandData, channel) {
+    if (trivData.eventFlag) {
+        Trivia.sendPM(src, "You cannot use /apropos during event games!", channel);
```

Comment:
I'll talk to Asca about it, but I think it was allowed for quickly searching for questions that needed fixing as they are found during games. It used to be disabled, but this restriction was removed around the time Miku was sTA iirc because some people complained about it. Since then, /markq has been added, so it may be okay to disable it again.